### PR TITLE
Bug 1129030 - Xfail TestKill_test_kill_multiple because of Bug 1125759

### PR DESCRIPTION
**Pull request by bebef1987**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[bebef1987 Florin Strugariu (bebef1987)](https://github.com/bebef1987) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=79404fddd51f)

File: tests/python/gaia-ui-tests/gaiatest/tests/unit/manifest.ini [hunk 1]
```diff
@@ -21,6 +21,9 @@ lan = true
 [test_contacts.py]
 [test_file_manager.py]
 [test_kill.py]
+[test_kill_multiple.py]
+# Bug 1125759 - Intermitent unit test TestKill_test_kill_multiple
+expected = fail
```
Comment (davehunt) [inline tests/python/gaia-ui-tests/gaiatest/tests/unit/manifest.ini]: If this is intermittent then we shouldn't mark it as expected to fail, we should disable it.

File: tests/python/gaia-ui-tests/gaiatest/tests/unit/test_kill.py [hunk 2]
```diff
@@ -15,21 +15,3 @@ def test_kill(self):
         app = self.apps.launch(Clock.name)
         self.apps.kill(app)
         self.check_no_apps_running()
-
-    def test_kill_multiple(self):
-        running_apps = []
-
-        for app in [Calendar.name, Clock.name]:
```
Comment (davehunt) [inline tests/python/gaia-ui-tests/gaiatest/tests/unit/test_kill.py]: Nit: Remove the Calendar import.

File: tests/python/gaia-ui-tests/gaiatest/tests/unit/test_kill_multiple.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+        for app in running_apps:
+            self.apps.launch(app.name)
+            time.sleep(1)
+            self.apps.kill(app)
+            time.sleep(1)
+
+        self.check_no_apps_running()
+
+
```
Comment (davehunt) [inline tests/python/gaia-ui-tests/gaiatest/tests/unit/test_kill_multiple.py]: Nit: Remove blank line.

**Comment by try-server-hook** [comment]
[bebef1987 Florin Strugariu (bebef1987)](https://github.com/bebef1987) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=36ef3238fb42)

**Comment by viorelaioia** [comment]
Lgtm

**Pull request merged**